mandatory/2-logic-error.js

```diff
@@ -1,16 +1,15 @@
 // The syntax for this function is valid but it has an error, find it and fix it.
 
 function trimWord(word) {
-  return wordtrim();
+  return word.trim();
 }
 
 function getWordLength(word) {
-  return "word".length();
+  return word.length;
 }
 
 function multiply(a, b, c) {
-  a * b * c;
-  return;
+      return a*b*c;
```

Comment:
what happened to the original spacing between a,b,c

Reply:
Thank you Jontaz77 for your comment. I'll review it and make the necessary changes.